### ISSUE TITLE
Nominate contributors to be added to velero-contributors team 

### ISSUE THREAD

**Issue by OrlinVasilev** [vmware-tanzu/velero#6545]
Let's use this issue to vote for new contributors to be officially added to @vmware-tanzu/velero-contributors team so they can review/comment/manage in general Issues and PRs
Please add your nominations so the maintainers can vote on them!

**Comment by OrlinVasilev** [comment]
@anshulahuja98

**Comment by OrlinVasilev** [comment]
@draghuram

**Comment by OrlinVasilev** [comment]
@kaovilai

**Comment by kaovilai** [comment]
@weshayutin

**Comment by reasonerjt** [comment]
@OrlinVasilev 
May I know more details in terms of `review/comment/manage in general Issues and PRs` ?

IMO Tthe permission to approve and merge PRs should be limited to maintainers.

@sseago @shubham-pampattiwar let's discuss it in the community meeting this week.

**Comment by OrlinVasilev** [comment]
Hi Daniel, as this is generally discussed here: https://github.com/vmware-tanzu/velero/blob/main/GOVERNANCE.md if we are missing something we can work it out in the process and define it better, I agree that PR merge is the maintainer's responsibility, but experienced contributors can review and suggest changed and improvements on the PRs and help with the issues(comment and close)

**Comment by kaovilai** [comment]
> experienced contributors can review and suggest changed and improvements on the PRs and help with the issues(comment and close)

With the exception of closing issues, we can already today provide approvals/suggested changes (gray checkmark) that do not result in allowing merge a particular PRs.

**Comment by kaovilai** [comment]
Recognition and permissions for things like labeling/milestones perhaps.

**Comment by kaovilai** [comment]
And closing duplicate issues.

**Comment by draghuram** [comment]
Permission to assign issues would be useful.

**Comment by reasonerjt** [comment]
@OrlinVasilev we've had a discussion in the community meeting today, and for the short term we decided rather than managing the team and the custom role on github, we'll just let developers ping the maintainers to close/assign issues/PRs. 

and per the governance doc, this"Contributor" role just reflects some behaviors of developers rather than mapping them to a set of permissions.  

In long term if we wanna make any change in the roles and permission we should first discuss and make sure changes are made to the governance doc and then make change to the permissions and members.

**Comment by OrlinVasilev** [comment]
Hi @reasonerjt, as discussed initially in the previous community meeting https://www.youtube.com/watch?v=XQkIGMjJEO0, that issue/discussion here, can trigger a change in the governance doc so we can better reflect the need, defining the contributor role with more details, can reflect positively to maintainers team in the following area:
- reduce maintainers' effort on reviewing PRs, by offloading some of the reviews to the contributors, merging will stay unchanged
- reduce the effort in terms of issue triaging
- gaining more technical knowledge for contributors
- fostering future maintainers
- public recognition to dedicated contributors by officially holding a role in the project!

**Comment by sseago** [comment]
@OrlinVasilev On this point in particular:
"reduce maintainers' effort on reviewing PRs, by offloading some of the reviews to the contributors, merging will stay unchanged"
Is there a specific proposal around what this means? Current requirement is 2 maintainer reviews. In order to offload reviews from maintainers, I'm assuming this would change in some way? Did you have anything in particular in mind?

Also, I just checked the governance doc. I thought that the two-maintaner-reviews requirement was there, but I don't see it. Perhaps we should make it explicit?

Here is our current process: "Before merging a PR, it must have two approving reviews by maintainers. Maintainers are allowed to merge."

One possibility to meet the offloading-reviews process is: "Before merging a PR, it must have two approving reviews from maintainers and contributors, at least one must come from a maintainer. Only maintainers are allowed to merge."

If we added the second bit above to the governance doc, then we'd need a contributor's group, and we'd need to change the auto-assign rules so that we still have 2 assigned reviewers, but instead of 2 maintainers, we'd have at least one maintainer, and one who could be a maintainer or contributor. I'm not sure how easy that is to do with github actions, though.
